test/inlining/InlineCallbacks.baseline

```diff
@@ -17,3 +17,11 @@ foo
 bar
 foo
 bar
+foo
+foo
+INLINING : Found callback def instr for call/apply target callback at	CallSite: 0	Caller: DispatchCall ( (#1.23), #24)

```

Comment:
Noticed that the actual inlining trace is not printed (as in when the call back target actually got inlined). Might be worthwhile to add that support.

Reply:
with -testtrace:inline it prints. I have this test running with just -testtrace:inlineCallbacks because of how noisy the other trace is. I don't really have a strong preference either way though.

Reply:
You could still keep the test running with -testtrace:inlinecallbacks, just call INLINE_CALLBACKS_TRACE with the right trace message when you are sure that callback target will be inlined. In my opinion, that will make this test more fit to catch regressions where callback target inlining may get inadvertently turned off.

Reply:
That makes sense. Added an extra trace.